storage/gluster-health-check

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+GLUSTER_PODS=$(oc get pods -l glusterfs-node=pod --all-namespaces -o wide | tail -n +2 | awk '{print $2}')
```

Comment:
Instead of tail/awk for these 3 commands. Would it not be better to use jsonpath?

Reply:
isn't that a command you have to install and isn't on most machines by default? Was trying to avoid having to install any specific packages. but could....

Reply:
no, oc supports jsonpath out of the box. you can just do: 
`oc get pods -l glusterfs-node=pod --all-namespaces -o jsonpath='{.items[].metadata.name}'`

Reply:
learn something new every day.

Reply:
fyi: https://kubernetes.io/docs/reference/kubectl/jsonpath/

Reply:
done.